### ISSUE TITLE
llama 3.2 1B instruct spinquant int 4 as .pte

### ISSUE THREAD

**Issue by v-prgmr** [facebookresearch/SpinQuant#16]
Hi everyone, 

Has anyone successfully created a .pte of llama 3.2 1B instruct spinquant from https://huggingface.co/meta-llama/Llama-3.2-1B-Instruct-SpinQuant_INT4_EO8/tree/main

Asking because I am facing issues trying to convert it using spinquant and also while trying to use the executorch export scripts. 

Thank you.

**Comment by kuantuna** [comment]
Yes, I went through [these steps](https://github.com/pytorch/executorch/tree/main/examples/models/llama#step-1-setup). Installed executorch and built relevant stuff, downloaded the folder from [here](https://huggingface.co/meta-llama/Llama-3.2-1B-Instruct-SpinQuant_INT4_EO8/tree/main), ran the `export_llama` script (the one specifically for spinquant) by replacing `LLAMA_QUANTIZED_CHECKPOINT` and `LLAMA_PARAMS` with the paths of the corresponding files downloaded. And then I obtained a .pte file.
